app/static/js/slcManagement.js

```diff
@@ -24,9 +28,49 @@ $(document).ready(function() {
 
       } else if ($(this).attr('data-content') == "In Progress") {
         return "This proposal has not been submitted for review."
+
+      } else if ($(this).attr('data-content') == "Imported") {
```

Comment:
This may not be the best status message for imported courses.

Reply:
Fixed